Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/callbacks/ShowNotificationCallBack.java

```diff
@@ -6,4 +6,5 @@
 
 public interface ShowNotificationCallBack {
     void showInAppNotification(JSONObject jsonObject, Context context);
+    void hideInAppNotification(String channelId);
```

Comment:
hide notification channelId is same as that of showNotification ?

Reply:
yes @ashishsingh101